frontend/lib/src/components/widgets/Multiselect/Multiselect.tsx

```diff
@@ -125,27 +125,22 @@ const Multiselect: FC<Props> = props => {
 
   const valueFromState = useMemo(() => {
     return value.map(i => {
-      const label = element.options[i]
+      const label = i
```

Comment:
does `i` refer to an option here? maybe rename to `option` or something more specific.